src/types/Person.ts

```diff
@@ -3,9 +3,10 @@ export interface Person {
   sex: string;
   born: number;
   died: number;
-  fatherName: string | null;
-  motherName: string | null;
+  fatherName?: string;
+  motherName?: string;
   slug: string;
   mother?: Person;
   father?: Person;
```

Comment:
The optional `mother` and `father` properties are of type `Person`, which could lead to circular references. Consider whether these properties are necessary or if a different structure might be more appropriate to avoid potential issues with serialization or cloning.